ComfortZoneApp/app/src/main/res/layout/fragment_input.xml

```diff
@@ -35,47 +35,85 @@
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="30dp"
                 android:textColor="@color/black"
                 android:textSize="18sp"
                 tools:text="Tuesday, Jun 21" />
 
             <LinearLayout
+                android:id="@+id/llLocation"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:layout_below="@+id/tvTime"
+                android:layout_marginTop="20dp"
+                android:gravity="center"
                 android:orientation="horizontal">
 
                 <ImageView
-                    android:id="@+id/imageView"
+                    android:id="@+id/imageView3"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:src="@drawable/ic_baseline_location_on_24"
-                    app:tint="@color/blue" />
+                    android:src="@drawable/ic_baseline_location_on_24" />
 
                 <TextView
                     android:id="@+id/tvLocation"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:gravity="center"
                     android:textColor="@color/black"
                     android:textSize="18sp"
                     tools:text="Menlo Park, CA" />
 
             </LinearLayout>
 
-
-
-
             <TextView
                 android:id="@+id/tvFeelsLike"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
+                android:layout_below="@+id/llLocation"
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="25dp"
                 android:textColor="@color/black"
                 android:textSize="50sp"
+                android:textStyle="bold"
                 tools:text="75" />
 
-        </LinearLayout>
+            <ImageView
+                android:id="@+id/imageView"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_alignParentBottom="true"
+                android:layout_centerHorizontal="true"
+                android:layout_marginBottom="0dp"
+                android:adjustViewBounds="true"
+                android:src="@drawable/ic_cloud_bottom" />
+
+            <TextView
+                android:id="@+id/tvTime"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tvDate"
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="10dp"
+                android:textColor="@color/black"
+                tools:text="5:34 PM" />
+
+            <TextView
+                android:id="@+id/textView8"
```

Comment:
Not sure which textview this refers to textView8. Its better to rename it wrt what UI element it refers to instead of numeric value.